tests/stress/stress_test.go

```diff
@@ -644,42 +644,83 @@ func TestSync_SubmitExtrinsicLoad(t *testing.T) {
 }
 
 func TestStress_SecondarySlotProduction(t *testing.T) {
-	numNodes := 3
-
-	nodes, err := utils.InitializeAndStartNodes(t, numNodes, utils.GenesisThreeAuths, utils.ConfigDefault)
-	require.NoError(t, err)
-
-	primaryCount := 0
-	secondaryCount := 0
-	otherCount := 0
-	for i := 1; i < 20; i++ {
-		fmt.Printf("%d iteration\n", i)
-		hash, err := utils.GetBlockHash(t, nodes[0], fmt.Sprintf("%d", i))
-		require.NoError(t, err)
+	testcases := []struct {
+		description  string
+		genesis      string
+		allowedSlots gosstypes.AllowedSlots
+	}{
+
+		{
+			description:  "with secondary vrf slots enabled",
+			genesis:      utils.GenesisTwoAuthsSecondaryVRF0_9_10,
+			allowedSlots: gosstypes.PrimaryAndSecondaryVRFSlots,
+		},
+		// TODO: We are not able to pickup the right value of AllowedSlots
+		// from these genesis file. It is always being read as PrimaryAndSecondaryVRFSlots.
+		// We have to fix that in order to properly test below cases.
+		// {
+		// 	description:  "with secondary plain slots enabled",
+		// 	genesis:      utils.GenesisTwoAuthsSecondaryPlain_0_9_10,
+		// 	allowedSlots: gosstypes.PrimaryAndSecondaryPlainSlots,
+		// },
+		// {
+		// 	description:  "with secondary slots disabled",
+		// 	genesis:      utils.GenesisTwoAuthsPrimary_0_9_10,
+		// 	allowedSlots: gosstypes.PrimarySlots,
+		// },
+	}
+	const numNodes = 2
+	for _, c := range testcases {
+		c := c // bypass scopelint false positive
```

Comment:
nit remove if you don't run tests in parallel. Also we don't use scopelint I think anymore.

```suggestion
```